src/samples/flow/iio/sol-flow-intel-minnow-max-linux_gt_3_17.json

```diff
@@ -64,6 +64,14 @@
            "buffer_size": "-1"
      },
      "type": "iio/thermometer"
+  },
+  {
+     "name": "MyProximity",
+     "options": {
+           "iio_device": "apds9960 create,i2c,platform/80860F41:05,0x39,apds9960 0",
+           "buffer_size": "-1"
```

Comment:
@edersondisouza explained to me at IRC, but I think the "0" (auto) logic is missing some pieces and should understand devices that do not support buffering. We'll fix that later, after that we could always go with the default value (0) and it should work, the buffer must be used only explicitly.